src/components/scores/export.cjsx

```diff
@@ -134,6 +136,7 @@ ScoresExport = React.createClass
         isWaiting={ScoresExportStore.isExporting(courseId) or tryToDownload}
         isFailed={ScoresExportStore.isFailed(courseId) or downloadHasError}
         failedProps={failedProps}
+        timeout={3600000}
```

Comment:
Why is the timeout 1 hour? And could/should all of these `3600000` be pulled together to the top of the file?